<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pylt</class>
 <widget class="QWidget" name="Pylt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pylt</string>
  </property>
  <widget class="QTabWidget" name="TabWiget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>240</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>1</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>F</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton:pressed
{
background-color:orange;

}

#pushButton:hover
{

background-color:orange;


}

</string>
     </property>
     <property name="text">
      <string> +</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>P</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_3:pressed
{
background-color: orange;

}
#pushButton_3:hover
{
background-color: orange;

}</string>
     </property>
     <property name="text">
      <string> -</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Скорость движения </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Положение по координате X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit:hover
{

background-color:yellow ;

}
#lineEdit
{

background-color:white ;

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit_2:hover
{

background-color:yellow ;

}
#lineEdit_2
{

background-color:white ;

}
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>2</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Состояние резаков</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_4{
     background-color:green;
}

#pushButton_4:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_5{
     background-color:green;
}

#pushButton_5:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_19">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_19
{

background-color:orange;
setText:&quot;All Go&quot;;

}
#pushButton_19:on
{
background-color:red;

}</string>
     </property>
     <property name="text">
      <string>All</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_20">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_20 {
	background-color: red;

    
}
#pushButton_4:off
{



}</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_6{
     background-color:green;
}

#pushButton_6:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_7{
     background-color:green;
}

#pushButton_7:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_8{
     background-color:green;
}

#pushButton_8:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_9{
     background-color:green;
}

#pushButton_9:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_10{
     background-color:green;
}

#pushButton_10:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_11{
     background-color:green;
}

#pushButton_11:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_12{
     background-color:green;
}

#pushButton_12:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_13{
     background-color:green;
}

#pushButton_13:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>31</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_14{
     background-color:green;
}

#pushButton_14:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>11</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_15 {
	background-color: green;

    
}

#pushButton_15:on{

     background-color: orange;
}
</string>
     </property>
     <property name="text">
      <string>12</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>3</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>P1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>P2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>P3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>P4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_17:pressed
{
background-color: orange;


}
#pushButton_17:hover
{
background-color: orange;

}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_18:pressed
{
background-color: orange;


}
#pushButton_18:hover
{
background-color: orange;

}</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit_3:hover
{

background-color:yellow ;

}
#lineEdit_3
{

background-color:white ;

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit_4:hover
{

background-color:yellow ;

}
#lineEdit_4
{

background-color:white ;

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit_5:hover
{

background-color:yellow ;

}
#lineEdit_5
{

background-color:white ;

}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#lineEdit_6:hover
{

background-color:yellow ;

}
#lineEdit_6
{

background-color:white ;

}</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
